install_my_pihole.sh

```diff
@@ -4,22 +4,20 @@
 
 ## FUNCTIONS
 function confirmName() {
-echo "So your full name is $1 $2?"
+  echo "So your full name is $1 $2?"
 }
 
 ## GLOBAL VARS
 FIRST_NAME=""
 LAST_NAME=""
 
 ## LOGIC
-echo "What first name?";
+echo "What is first name?";
 read;
-FIRST_NAME=${REPLY}
+rememberFirstName $REPLY
 
-rememberFirstName $FIRST_NAME
-
-echo "What last name?";
+echo "What is last name?";
 read;
-LAST_NAME=${REPLY}
+rememberLastName $REPLY
+
 
```

Comment:
To pass FIRST_NAME and LAST_NAME to confirmName() you will want the following:

`confirmName $FIRST_NAME $LAST_NAME`